go/mysql/capabilities/capability.go

```diff
@@ -48,6 +48,8 @@ const (
 	PerformanceSchemaDataLocksTableCapability                           // supported in MySQL 8.0.1 and above: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html
 	InstantDDLXtrabackupCapability                                      // Supported in 8.0.32 and above, solving a MySQL-vs-Xtrabackup bug starting 8.0.29
 	ReplicaTerminologyCapability                                        // Supported in 8.0.26 and above, using SHOW REPLICA STATUS and all variations.
+	RestrictFKOnNonStandardKey                                          // Supported in 8.4.0 and above, restricts usage of non-standard indexes for foreign keys.
+	BinaryLogStatus                                                     // Supported in 8.4.0 and above, uses SHOW BINARY LOG STATUS
```

Comment:
I believe this is 8.2.0 and above, not 8.4.0 and above?

Reply:
See the SQL Syntax Notes section of https://dev.mysql.com/doc/relnotes/mysql/8.2/en/news-8-2-0.html.

Reply:
@GrahamCampbell WIP, haven't audited the final additions yet. For Vitess we only really care about MySQL LTS releases anymore. I'll fix once it all works.

Reply:
Fixed up!